src/add-remove.js

```diff
@@ -0,0 +1,99 @@
+// eslint-disable-next-line import/no-cycle
+import checkboxesEvent, {
+  list, save, fixIndex, setList, remove,
+} from './status-update';
+
+const todoList = document.getElementById('todo-list');
+export const items = document.getElementsByClassName('task-item');
```

Comment:
- [ ] Nice job for adding the _trash icon_ for deleting each task. However, the _trash icon_ **does not remove** an item when clicked on. Kindly ensure to improve upon this aspect.

**Your design:**
![image](https://user-images.githubusercontent.com/12745474/134746301-eaad69eb-557f-46dd-89c3-ec056492bcd8.png)